polaris/benchmark/_base.py

```diff
@@ -166,19 +166,18 @@ def _validate_main_metric(cls, v):
     def _validate_split(cls, v, info: ValidationInfo):
         """
         Verifies that:
-          1) There is at least two, non-empty partitions
+          1) There are no empty test partitions
           2) All indices are valid given the dataset
           3) There is no duplicate indices in any of the sets
           3) There is no overlap between the train and test set
         """
 
-        # There is at least two, non-empty partitions
-        if (
-            len(v[0]) == 0
```

Comment:
**Suggestion**: I could see an empty train set be confusing, so for now I suggest we clearly log on benchmark creation that this is a test set only benchmark that will return an empty train set. E.g.:

```py
if len(v[0]) == 0: 
    logger.info("This benchmark only specifies a test set. It will return an empty train set in `get_train_test_split()`")
```